seed/tests/test_api.py

```diff
@@ -32,32 +32,6 @@ def test_get_api_endpoints_utils(self):
         for url, fn in res.items():
             self.assertTrue(fn.is_api_endpoint)
             self.assertTrue(url.startswith('/'))
-            # these urls are generated by the DRF router and with /?
```

Comment:
@nllong and/or @axelstudios 
I removed this thinking it was stale. ([It's from the initial commit of SEED.](https://github.com/SEED-platform/seed/blame/2d924db8d16c2052034d5b0e3889c53b5b2b3f24/seed/tests/test_api.py#L94)).

Before this removal, we just kept adding URIs to the tuple below. It seemed unnecessary, but let me know if you all have more historical background on this and why we've kept it.

Reply:
I'll be glad to see that gone if it's no longer needed

Reply:
After taking a look myself, I'm not sure we really need it anymore. it's used by the `get_api_schema` view, at `/api/v2/schema`. If we're deprecating that view it might be nice to clean all of that stuff up

Reply:
> After taking a look myself, I'm not sure we really need it anymore. it's used by the `get_api_schema` view, at `/api/v2/schema`. If we're deprecating that view it might be nice to clean all of that stuff up

That endpoint might be cleaned up (/deprecated) with #2254.